lib/l1_sender/src/config.rs

```diff
@@ -1,55 +1,20 @@
 use alloy::consensus::constants::GWEI_TO_WEI;
 use serde::{Deserialize, Serialize};
-use smart_config::value::SecretString;
-use smart_config::{DescribeConfig, DeserializeConfig};
```

Comment:
Let's get rid of smart-config deps in all `Cargo.toml`s except for bin -- to make sure we deleted all irrelevant entries